### DISCUSSION TITLE
Magic Link Expiration Time for Email Logins

### DISCUSSION THREAD

**Discussion by MohanaMeher** [supabase/supabase#29298] [Feature Requests] [answered]
I am currently using email logins with Magic Links for authentication. By default, the confirmation link expires after 60 minutes, but my application requires these links to be valid for 72 hours.

Is there a way to extend or customize the expiration time for Magic Links? I couldn't find any settings to adjust the expiration duration. If it's not currently supported, are there plans to implement this feature, or would it require a custom solution?

**Comment by GaryAustin1** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/d7551736-ed92-45e1-8c7b-0336b27b7cb3)
email provider

**Reply by GaryAustin1** [reply]
But 24 hours is the limit.

**Reply by MohanaMeher** [reply]
Thanks so much @GaryAustin1 
Appreciate your help
